src/pages/kb/data-sources/bigquery-setup.md

```diff
@@ -8,17 +8,19 @@ toc: true
 
 ### Data Source Setup Screen
 
-When setting up the BigQuery data source, only the **Project ID** and **JSON Key File** are required. If you don't have a JSON key file then you will need to create a new service account (details below).
+To set up the BigQuery data source only the **Project ID** and **JSON Key File** are required by Redash. Occasionally, BigQuery will require that you include a **Processing Location** as well. You can get a key file when you create a new service account with Google (details below). 
 
 ![](/assets/images/docs/gitbook/bigquery_mandatories.PNG)
 
-+ The **Load Schema** tickbox lets you disable Redash from fetching your BigQuery schema. You may want to do this if your BigQuery instance contains more than \~2500 tokens. Application performance can deteriorate above this threshold because the schema is loaded into your browser on the Query Editor screen.
++ If your database schema exceeds \~2500 tokens, untick the **Load Schema** box to stop Redash from loading it to the Query Editor screen. Many browsers will slow down or crash if the schema is too big.
 
-+ Since BigQuery 2.0, you can use either BigQuery SQL syntax (now called Legacy SQL) or [Standard SQL Syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql). Redash supports both (Legacy SQL is the default) but this preference applies at the Data Source-level by toggling the **Use Standard SQL** box. Your selection here is passed to BigQuery along with your query text. If some of your queries use Legacy SQL and others use Standard SQL, you can create two data sources.
++ Since BigQuery 2.0, you can use either BigQuery SQL syntax (now called Legacy SQL) or [Standard SQL Syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql). Redash supports both, but Legacy SQL is the default.  but this preference applies at the Data Source-level by toggling the **Use Standard SQL** box. Your selection here is passed to BigQuery along with your query text. If some of your queries use Legacy SQL and others use Standard SQL, you can create two data sources.
```

Comment:
```suggestion
+ Since BigQuery 2.0, you can use either BigQuery SQL syntax (now called Legacy SQL) or [Standard SQL Syntax](https://cloud.google.com/bigquery/docs/reference/standard-sql/migrating-from-legacy-sql). Redash supports both, but Standard SQL is the default.  This preference applies at the Data Source-level by toggling the **Use Standard SQL** box. Your selection here is passed to BigQuery along with your query text. If some of your queries use Legacy SQL and others use Standard SQL, you can create two data sources.
```

(Standard is the default)

Reply:
Good catch. [Here's](https://github.com/getredash/redash/blob/cab011def96158adb89c4a00290e1a20ef3f4bd0/redash/query_runner/big_query.py#L113) the line that confirms that.